data/scenarios/Fun/GoL.yaml

```diff
@@ -0,0 +1,71 @@
+name: Conway's Game of Life
+description: A basic implementation of the rules of the Game of Life.
+robots:
+  - name: base
+    display:
+      attr: robot
+      char: 'Ω'
+    loc: [0,0]
+    dir: [1,0]
+    devices:
+      - treads
+      - compass
+      - grabber
+      - clock
+      - logger
+    inventory:
+      - [1000, rock]
+  - name: cell
+    system: true
+    dir: [0,1]
+    display:
+      invisible: true
+    inventory:
+      - [1, rock]
+    program: |
+      def forever = \c. c; forever c; end;
+      def cnt = \x.
+        if (x == inl ()) {0} {1}
+      end;
+      def count3 =
+        h <- scan down;
+        f <- scan forward;
+        b <- scan back;
+        return (cnt h + cnt f + cnt b)
+      end;
+      forever (
+        h <- scan down;
+        alive <- return (h != inl ());
+        n1 <- count3;
+        turn left; move; turn right; n2 <- count3;
+        turn right; move; move; turn left; n3 <- count3;
+        turn left; move; turn right;
```

Comment:
@byorgey you know, if `as` was special in `atomic`, you could check all of this hypothetically in one tick _and_ place/grab the rock. :wink:

Reply:
:laughing:  Haha, that definitely seems like an abuse of `atomic`!